### PR TITLE
update contribution doc and PR template

### PR DESCRIPTION
**Pull request by mnabian**
# Modulus Pull Request

## Description

closes https://github.com/NVIDIA/modulus/issues/114

## Checklist

- [x] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/modulus/blob/main/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.
- [x] The [CHANGELOG.md](https://github.com/NVIDIA/modulus/blob/main/CHANGELOG.md) is
up to date with these changes.

### CODE REVIEW

**Comment by NickGeneva** [comment]
Oops: https://github.com/NVIDIA/modulus/pull/128/files

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ to merge the changes from a branch of the fork into a selected branch of upstrea
     - Atleast one Modulus engineer will be assigned for the review.
 
 4. The PR will be accepted and the corresponding issue closed after adequate review and
-testing has been completed.
+testing has been completed. Note that every PR should have an open issue.
```
Comment (NickGeneva) [inline CONTRIBUTING.md:51]: "Note that every PR should correspond to an open issue and should be linked on Github."

**Review by NickGeneva — APPROVED** [review]

**Review by ktangsali — APPROVED** [review]

**Pull request merged**